src/Config/EventConfigurator.php

```diff
@@ -59,6 +71,23 @@ private function isCallable($definition): bool
             return true;
         }
 
-        return is_array($definition) && array_keys($definition) === [0, 1] && in_array($definition[1], get_class_methods($definition[0]) ?? [], true);
+        if (
+            is_array($definition)
+            && array_keys($definition) === [0, 1]
+            && is_string($definition[0])
+            && $this->container->has($definition[0])
+        ) {
+            try {
+                $object = $this->container->get($definition[0]);
+
+                return method_exists($object, $definition[1]);
+            } catch (NotFoundExceptionInterface|ContainerExceptionInterface $exception) {
+                $message = "Could not instantiate event listener or listener class has invalid configuration.";
```

Comment:
It might be better to remove $message and pass the message direct to Exception ?